domain/domain.go

```diff
@@ -648,7 +648,11 @@ func (do *Domain) Init(ddlLease time.Duration, sysFactory func(*Domain) (pools.R
 	if ebd, ok := do.store.(tikv.EtcdBackend); ok {
 		if addrs := ebd.EtcdAddrs(); addrs != nil {
 			cfg := config.GetGlobalConfig()
+			// silence etcd warn log
```

Comment:
It is best to describe in detail the reason for changing to the error level, otherwise, we may forget in the future.

Reply:
Sure, I add detail  and put issue link in description.